api/openapi.yaml

```diff
@@ -324,14 +324,12 @@ components:
         - title
         - status
         - detail
-      example: |
```

Comment:
@tothandras please take a look at these fixes. According to the tool, this needs to be an object.